CGAL_ipelets/demo/CGAL_ipelets/CMakeLists.txt

```diff
@@ -97,7 +97,7 @@ if(IPE_FOUND AND IPE_VERSION)
     cgal_add_compilation_test(CGAL_${IPELET})
   endforeach(IPELET)
   if(CGAL_Core_FOUND)
-    target_link_libraries(CGAL_cone_spanners PRIVATE CGAL::CGAL_Core
+    target_link_libraries(CGAL_cone_spanners PRIVATE CGAL::CGAL CGAL::CGAL_Core
```

Comment:
C'est pas plutôt `simple_triangulation` qui aurait besoin d'être linké avec `CGAL::CGAL` ?

Reply:
And the translation is?

Reply:
Sorry. I have edited my comment.